Yoast/wordpress-seo
# Change advanced metabox noindex question to be human readable

## Summary

This PR can be summarized in the following changelog entry:

* The advanced tab on the edit post page now asks "Allow search engines to show this Post in search results?" instead of having a heading "Meta Robots", which was quite non-understandable for non-SEO's. Similar changes have been made to the `follow` / `nofollow` setting.

## Test instructions

This PR can be tested by following these steps:

* Go to the edit post page for any post, see the difference on the advanced tab of the Yoast SEO metabox.
* Go to the edit post page for any _page_, see the difference on the advanced tab of the Yoast SEO metabox.


## Quality assurance
